Resources/Prototypes/Catalog/VendingMachines/Inventories/seeds.yml

```diff
@@ -39,5 +39,8 @@
     TowercapSeeds: 5
     WheatSeeds: 5
     WatermelonSeeds: 5
+  contrabandInventory:
+    NettleSeeds: 2
```

Comment:
Considering there have been talks about buffing deathnettle, I would not make it more available like this.

Reply:
I... did not know that. Amantia is pretty strong as it is, so probably not that. Off the top of my head, I should just swap this for Tobbaco or Bungo seeds, basically just one of the lesser-used exotics. Probably Bungo.

Reply:
Swapped for Bungo Seeds. While its pits can make a decent poison at .2u/sec at 1.8 poison/sec, it's much weaker than Fly Amantia and the upcoming Nettle, and literally the only other choice is tobacco. I think this fits what I was going for with Nettle.